src/ConfirmDeletePage.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export function ConfirmDeletePage(props) {
+
+    return (
+        <div>
+            <h1 id="deleteList">Are you sure you want to delete {props.listName}?</h1>
+            <input type="button" value="Cancel"
```

Comment:
Is this enough information for someone using a screen reader to know what these buttons are for?